src/pages/patientView/clinicalInformation/getClinicalInformationData.ts

```diff
@@ -0,0 +1,106 @@
+import queryString from 'query-string';
+import { chain } from 'underscore';
+import CBioPortalAPI from "shared/api/CBioPortalAPI";
+import { getTreeNodesFromClinicalData } from './PDXTree';
+import {ClinicalDataSummary} from "../../../shared/api/CBioPortalAPI";
+import {SampleSummary} from "../../../shared/api/CBioPortalAPI";
+//import sampleQuery from 'shared/api/mock/Samples_query_patient_P04.json';
+
+
+type TODO = any;
+
+/*
+ * Transform clinical data from API to clinical data shape as it will be stored
+ * in the store
+ */
+function transformClinicalInformationToStoreShape(patientId: string, studyId: string, clinicalDataPatient: Array<ClinicalDataSummary>, clinicalDataSample: Array<ClinicalDataSummary>) {
+    const patient = {
```

Comment:
I think it should be in a new file like this one: https://github.com/adufilie/cbioportal-frontend/blob/ba8cb3b736aacb22d033a6d9b2d1333d094d6878/src/shared/api/cbioportal-types.d.ts